src/onboarding/Start.js

```diff
@@ -151,14 +160,18 @@ class StartContent extends React.PureComponent {
       <React.Fragment>
         <Title>
           <Text size="great" weight="bold" color={theme.textDimmed}>
```

Comment:
This should be `24px` on small screens, but since this size doesn’t exist, maybe we could use [`22px` (`xxlarge`)](https://github.com/aragon/aragon-ui/blob/master/src/utils/styles/font.js#L8) to avoid adding another size? /cc @jounih